i18n/src/parser.js

```diff
@@ -33,7 +46,12 @@ function parsePattern(pattern) {
       positive = patternParts[0],
       negative = patternParts[1];
 
-  var positiveParts = positive.split(DECIMAL_SEP),
+  // The parsing logic below assumes that there will always be a DECIMAL_SEP in the pattern.
```

Comment:
I'm a bit confused by this. Does it mean if a locale doesn't have a DECIMAL_SEP, we add it after the last ZERO, which means it will be discarded later (because nothing comes after it)? So we only add it for the sake of parsing?